src/sentry/static/sentry/app/components/dataExport.tsx

```diff
@@ -65,15 +69,15 @@ class DataExport extends React.Component<Props, State> {
     return (
       <Feature features={['data-export']}>
         {inProgress && dataExportId ? (
-          <Tooltip title={TooltipMessages.progress}>
+          <Tooltip title="Wait right there. We'll email you.">
```

Comment:
@leeandher If we are going to email them, why should we tell them to wait there? We should tell them they can do something else since we'll email them.

Reply:
That was one of the options Molly suggested, here they are if you wanna pick another!
 - Sit tight. We'll email you when it's finished.
 - Wait right there. We'll email you.
 - You can get on with your life. We'll email you when your data's ready.
 - Now, get back to whatever you were doing before. We'll email you.

Reply:
I like: `You can get on with your life. We'll email you when your data's ready.`

Reply:
Gonna change it to the fourth one :)

Reply:
haha, ok that one is also good

Reply:
Woops lol I hadn't refreshed! I just picked one out of the blue

Reply:
Okay updating it to that one, I like it too -- Thanks!!